docs/job_crd.adoc

```diff
@@ -4,8 +4,8 @@
 [id="{p}-api-reference"]
 == API Reference
 
-:minio-image: https://hub.docker.com/r/minio/minio/tags[minio/minio:RELEASE.2024-03-15T01-07-19Z]
-:kes-image: https://hub.docker.com/r/minio/kes/tags[minio/kes:2024-03-13T17-52-13Z]
+:minio-image: https://hub.docker.com/r/minio/minio/tags[minio/minio:RELEASE.2024-05-01T01-11-10Z]
+:kes-image: https://hub.docker.com/r/minio/kes/tags[minio/kes:2024-04-12T13-50-00Z]
```

Comment:
I think we should also fix the `mc` image to a specific version (it's now set to latest). See also https://github.com/minio/operator/pull/2173#discussion_r1648645037.

Reply:
mmmm, that is true, we are just pulling the latest MC and adding it on the crd doc, let me specify the mc version in the release.sh script, in a next PR

Reply:
set a specific mc image is covered in this PR @ramondeklein https://github.com/minio/operator/pull/2187